packages/ui/src/composables/useWeb3.js

```diff
@@ -81,18 +81,13 @@ export function useWeb3Witmon () {
           CONTRACT_ADDRESS
         )
         const from = (await requestAccounts(web3))[0]
-        const creatureData = await contract.methods
-          .getCreatureData(player.index)
-          .call()
-        if (creatureData) {
-          return creatureData
+        const data = await contract.methods.getData(player.index).call()
+        if (data) {
+          return data
         }
```

Comment:
Can we directly return data to avoid having an async function without a return?

```suggestion
        return data
```